diff --git a/index.js b/index.js
index d7ed315f898655019cc2e56be5e75352db5889ee..de32b536ccb3a26c98f27827aa741c825a5cbe48 100644
--- a/index.js
+++ b/index.js
@@ -50,7 +50,7 @@ addIntBindings(8);
 addIntBindings(16);
 addIntBindings(32);
 
-twoToThe32 = Math.pow(2, 32);
+var twoToThe32 = Math.pow(2, 32);
 
 // 64 bit bindings require special care
 var read64 = function(unsigned){return function(buff, endian){
